migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go

```
@@ -0,0 +1,133 @@
+	s.db = pghelper.ForT(s.T(), false)
+	pgutils.CreateTableFromModel(s.ctx, s.db.GetGormDB(), newSchema.CreateTableReportConfigurationsStmt)
+	s.gormDB = s.db.GetGormDB()
+	s.gormDB = s.gormDB.WithContext(s.ctx).Table("report_configurations")
+	s.snapshotgormdB = s.db.GetGormDB()
+	s.snapshotgormdB = s.snapshotgormdB.WithContext(s.ctx).Table("report_configurations_notifiers")
+
+	notifierdB := s.db.GetGormDB()
+	notifierdB = notifierdB.WithContext(s.ctx).Table("notifiers")
+	ret := fixtures.GetValidReportConfiguration()
+	ret.Id = reportID
+	notifierID := uuid.NewV4().String()
+	ret.NotifierConfig = &storage.ReportConfiguration_EmailConfig{
+		EmailConfig: &storage.EmailNotifierConfiguration{
+			NotifierId:   notifierID,
+			MailingLists: []string{"foo@yahoo.com"},
+		},
+	}
+	ret.LastRunStatus = &storage.ReportLastRunStatus{
+		ReportStatus: storage.ReportLastRunStatus_SUCCESS,
+	}
+
+	ret.LastSuccessfulRunTime = timestamp.Now().GogoProtobuf()
+	converted, err := newSchema.ConvertReportConfigurationFromProto(ret)
+	s.Require().NoError(err)
+	convertedReportConfigs := []*newSchema.ReportConfigurations{converted}
+	err = s.gormDB.Clauses(clause.OnConflict{UpdateAll: true}).Model(newSchema.CreateTableReportConfigurationsStmt.GormModel).Create(&convertedReportConfigs).Error
+	s.Require().NoError(err)
+
+	notifierProto := &storage.Notifier{
+		Id: notifierID,
+	}
+
+	notifier, err := newSchema.ConvertNotifierFromProto(notifierProto)
+	notifiers := []*newSchema.Notifiers{notifier}
+	s.Require().NoError(err)
+	err = notifierdB.Clauses(clause.OnConflict{UpdateAll: true}).Model(newSchema.CreateTableNotifiersStmt.GormModel).Create(&notifiers).Error
+	s.Require().NoError(err)
+}
+
+func (s *migrationTestSuite) TearDownSuite() {
+	s.db.Teardown(s.T())
+}
+
+func (s *migrationTestSuite) TestMigration() {
+	dbs := &types.Databases{
+		GormDB:     s.db.GetGormDB(),
+		PostgresDB: s.db.DB,
+		DBCtx:      s.ctx,
+	}
+
+	migration.Run(dbs)
+
+	configs, _ := s.gormDB.Rows()
```

After the respective loops, you should check `.Err()` on `configs` and `snapshots`.  You should also not ignore the error on the call to `Rows()`.  So you should do `configs, err := s.gormDB.Rows()` and then `s.Require().NoError(err)`